src/pages/page-names.constants.ts

```diff
@@ -45,3 +45,22 @@ export const CAT_BE = {
   VEHICLE_CHECKS_MODAL: 'VehicleChecksCatBEModal',
   VIEW_TEST_RESULT_PAGE:'ViewTestResultCatBEPage',
 };
+
+export const CAT_C = {
+  BACK_TO_OFFICE_PAGE: 'BackToOfficeCatCPage',
+  COMMUNICATION_PAGE: 'CommunicationCatCPage',
+  DEBRIEF_PAGE: 'DebriefCatCPage',
+  HEALTH_DECLARATION_PAGE: 'HealthDeclarationCatCPage',
+  OFFICE_PAGE: 'OfficeCatCPage',
+  PASS_FINALISATION_PAGE: 'PassFinalisationCatCPage',
+  REVERSE_DIAGRAM_PAGE: 'ReverseDiagramCatCPage',
+  TEST_REPORT_PAGE: 'TestReportCatCPage',
+  WAITING_ROOM_PAGE: 'WaitingRoomCatCPage',
+  WAITING_ROOM_TO_CAR_PAGE: 'WaitingRoomToCarCatCPage',
+  POST_DEBRIEF_HOLDING_PAGE: 'PostDebriefHoldingCatCPage',
+  REKEY_REASON_PAGE: 'RekeyReasonCatCPage',
+  REKEY_UPLOAD_OUTCOME_PAGE: 'RekeyUploadOutcomePage',
```

Comment:
Is there no cat c version of the rekey upload outcome page?

Reply:
That should not even be in any category, as it's not a category specific page. let's solve that later